### ISSUE TITLE
Error using cloudCache related to `fun` argument

### ISSUE THREAD

**Issue by tati-micheletti** [PredictiveEcology/reproducible#98]
When trying the following 
```
cloudFolderID <- "https://drive.google.com/open?id=1PoEkOkg_ixnAdDqqTQcun77nUvkEHDc0"
testingRTM <- cloudCache(prepInputs, url = "https://drive.google.com/open?id=1fo08FMACr_aTV03lteQ7KsaoN9xGx1Df", targetFile = "RTM.tif", destinationPath = tempdir(), useCloud = TRUE, cloudFolderID = cloudFolderID)
```
it returns 
```
Error in .guessAtTargetAndFun(targetFilePath, destinationPath, filesExtracted = filesExtr,  : fun must be a character string, not the function" 
```
When passing `fun = "raster::raster"` it works:
```
testingRTM <- cloudCache(prepInputs, url = "https://drive.google.com/open?id=1fo08FMACr_aTV03lteQ7KsaoN9xGx1Df", targetFile = "RTM.tif", destinationPath = tempdir(), useCloud = TRUE, cloudFolderID = cloudFolderID, fun = "raster::raster")
```

**Comment by tati-micheletti** [comment]
I would guess this happened after the latest changes in `cloudCache`, as it was working a few days ago without a problem.

**Comment by tati-micheletti** [comment]
Should be fixed with PR #99
